developer/src/tike/debug/Keyman.System.Debug.DebugEvent.pas

```diff
@@ -360,6 +360,13 @@ function TDebugEventList.AddStateItems(
 
   AddDebugItem(debug, debugkeyboard, vk, modifier_state);
 
+  if action._type = KM_CORE_IT_INVALIDATE_CONTEXT then
+  begin
+    // We always ignore invalidate context which can come when a frame key is
```

Comment:
lgtm.
Is this the same behavior on a non-debug event?

Reply:
No, the debugger specifically ignores the invalidate context event because it always tracks the context, including markers. It's the only app that ever knows about markers, so it's the only app that can ignore the event.